napari/_qt/qt_viewer.py

```diff
@@ -573,10 +574,15 @@ def toggle_console_visibility(self, event=None):
         )
 
     def show_key_bindings_dialog(self, event=None):
-        dialog = QtAboutKeyBindings(
-            self.viewer, self._key_map_handler, parent=self
-        )
-        dialog.show()
+        if self._key_bindings_dialog is None:
+            self._key_bindings_dialog = QtAboutKeyBindings(
+                self.viewer, self._key_map_handler, parent=self
+            )
+        # make sure the dialog is shown
+        self._key_bindings_dialog.show()
+        # make sure the the dialog gets focus
+        self._key_bindings_dialog.raise_()  # for macOS
+        self._key_bindings_dialog.activateWindow()  # for Windows
```

Comment:
> We've tended to use explicit references to our Qt elements so, something like a self._key_bindings_dialog on the QtViewer which would start as None and then get set. 

I like this better, as it made it obvious that there should only be one dialog window. I don't think being allowed to open a bunch of these windows really makes sense.

Reply:
nice, agreed!